src/Sentry/SentrySdk.cs

```diff
@@ -240,6 +240,17 @@ public void Dispose()
     [DebuggerStepThrough]
     public static IDisposable PushScope<TState>(TState state) => CurrentHub.PushScope(state);
 
+    /// <summary>
+    /// Restores a previously saved/cloned scope to the top of the stack
+    /// </summary>
+    public static void RestoreScope(Scope savedScope)
+    {
+        if (CurrentHub is IHubEx hub)
```

Comment:
This type checks are odd, and if the type isn't what we expect we'll just not observe the behavior we expect?
How about a `else debug.log` at least?

Reply:
I removed from the public interfaces/sdk entirely...